community/server/src/docs/ops/server-installation.asciidoc

```diff
@@ -61,7 +61,9 @@ If you see this error simply install Neo4j into a different location.
    * Select the appropriate Zip distribution.
 2. Right-click the downloaded file, click Extract All.
    * Refer to the top-level extracted directory as: +NEO4J_HOME+
-3. Consult <<powershell>> for how to start or install Neo4j.
+3. Change directory to: `$NEO4J_HOME`
```

Comment:
* Is the top-level extracted directory referred to as `NEO4J_HOME` or `$NEO4J_HOME`? Actually, it's not clear to me why we need to have a convention for how to refer to that directory. Can't we just say "Change directory to the top-level extracted directory", or similar?
* Prefer using italics rather than monospace (underscore vs backtick/plus) for filenames and paths. (`bin/neo4j console` is correctly marked as monospace/code however, since it is a command to be executed.)
* The colons at "directory as: NEO4J_HOME", "directory to: $NEO4J_HOME", and "Run: bin/neo4j consle" are superfluous.